### DISCUSSION TITLE
Per-image unique URLs in public link view

### DISCUSSION THREAD

**Discussion by Penknife0915** [ente-io/ente#566] [Enhancements]
The current public link view lacks a way for viewers to share or bookmark specific images because the URL does not update when an image is expanded from a thumbnail.

To address this, IMHO, two features could be considered: first, enabling the ability to long-press / right-click on thumbnails within the public link to copy the direct link for the desired image; second, updating the URL to the specific image in its expanded view.

Implementing these features would allow users to reshare and bookmark specific images from the public link view, a functionality that is currently missing.

**Comment by aakankshabhende** [comment]
@vishnukvmd I would like to work on this issue. Could you please assign me this?
